docs/generate-openapi.md

```diff
@@ -0,0 +1,189 @@
+# Generate OpenAPI from tests
+![GitHub Repo stars](https://img.shields.io/github/stars/opticdev/optic?style=social) ![GitHub contributors](https://img.shields.io/github/contributors-anon/opticdev/optic?style=social) ![npm](https://img.shields.io/npm/dm/@useoptic/openapi-io?style=social) ![license](https://img.shields.io/github/license/opticdev/optic?style=social)
+
+
+API tests contain a lot of undocumented information about how your API behaves. Optic leverages your existing tests to capture and persist that information to an OpenAPI specification. Optic's capture feature works by observing traffic that passes through a locally-running proxy and uses it to keep your OpenAPI specification accurate. When new endpoints are added or existing API behavior changes, Optic updates the OpenAPI spec to match the current API behavior.
+
+**Connect your tests, get accurate docs!**
+
+https://github.com/opticdev/optic/assets/5900338/d4e740b8-7ee2-4451-aee5-3ec062d5162b
+
+
+## Install
+```bash
+npm install -g @useoptic/optic
+```
+
+## Quick Start
+
+You can quickly kick the tires with our example API. 
+
+```bash
+git clone https://github.com/opticdev/bookstore-example.git
+cd bookstore-example
+optic capture openapi.yml
+```
+
+## Integrating Optic with your project
+
+**1. Connect your tests.** Tell Optic how to run your tests in the `optic.yml` config file.
+
+```yaml
+# optic.yml
+capture:
+  openapi.yml:
+    server:
```

Comment:
should we have a start command here?

Reply:
oops, yeah, either remove the server line, or add some made up command, `yarn run start` or whatever. depends on how this imaginary scenario works, do the tests start a server themselves, or is that a separate explicit step?